tests/suites/os/suite.js

```diff
@@ -261,39 +261,6 @@ module.exports = {
 		// Unpack OS image .gz
 		await this.os.fetch();
 
-		// If this is a flasher image, and we are using qemu, unwrap
```

Comment:
This change is also present on #2946, do you mean to remove this from here and transfer? Or close that PR?

Reply:
@vipulgupta2048 Good catch, this PR wasn't meant to include that.